src/commons/services/utils.js

```diff
@@ -679,5 +678,23 @@
             });
         };
 
+        vm.glusterBrickMapping = function(data, cluster) {
```

Comment:
@cloudbehl Start creating factories now. Don't add methods in utils. Its already very lengthy file.